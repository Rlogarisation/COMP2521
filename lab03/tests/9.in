29 17 11 13 23 19 41 31 37 43