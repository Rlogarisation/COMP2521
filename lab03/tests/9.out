Tree:
        29
        / \
       /   \
      /     \
     /       \
    17       41
   / \       / \
  /   \     /   \
 /     \   31   43
11     23   \
 \     /    37
 13   19

#nodes = 10,  #leaves = 4
Infix  : 11 13 17 19 23 29 31 37 41 43 
Prefix : 29 17 11 13 23 19 41 31 37 43 
Postfix: 13 11 19 23 17 37 31 43 41 29 
ByLevel: 29 17 41 11 23 31 43 13 19 37 
